### ISSUE TITLE
Enable sending multiple events concurrently

### ISSUE THREAD

**Issue by abbi-gaurav** [kyma-incubator/varkes#182]
As a developer, I should be able to trigger more than one event and also specify concurrency.
e.g. I should be able to specify sending 100 events with a concurrency factory of 2.

This should in parallel send 2 * 100 events.

**Comment by a-thaler** [comment]
Really nice to have feature, unfortunately, I don't see that we will do it caused by missing man power :(
